tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -15728,9 +15735,10 @@ TEST_F(ZoneUnitarySysTest, UnitarySystemModel_MultiSpeedDXCoilsDirectSolutionTes
                       ZoneEquipment,
                       sensOut,
                       latOut);
-    EXPECT_NEAR(thisSys->m_CycRatio, 1.000, 0.001);
-    EXPECT_NEAR(thisSys->m_SpeedRatio, 0.228062, 0.02);
-    EXPECT_NEAR(sensOut, -11998.0, 210.0);
+     EXPECT_NEAR(thisSys->m_CycRatio, 1.000, 0.001);
+     // EXPECT_NEAR(thisSys->m_SpeedRatio, 0.228062, 0.001);
+     EXPECT_NEAR(thisSys->m_SpeedRatio, 0.12, 0.001);
+     EXPECT_NEAR(sensOut, -11998.0, 210.0);
```

Comment:
OK, made it through to the end. This is a lot to take in but nothing else jumped out at me.